src/Basque/Mews.Fiscalizations.Basque/Dto/Bizkaia/Header/BizkaiaHeaderData.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Mews.Fiscalizations.Basque.Dto.Bizkaia.Header;
+
+public class BizkaiaHeaderData
+{
+    [JsonPropertyName("con")]
+    public string With { get; set; }
+    
+    [JsonPropertyName("apa")]
+    public string Section { get; set; }
+
+    [JsonPropertyName("inte")]
+    public Inte Inte { get; set; }
+
+    [JsonPropertyName("drs")]
+    public Drs Drs { get; set; }
+}
+
+public class Drs
+{
+    [JsonPropertyName("mode")]
+    public string Mode { get; set; }
+
+    [JsonPropertyName("ejer")]
+    public string FiscalYear { get; set; }
```

Comment:
If its only the year, then this can be an int

and any idea what should we provide here exactly? if it's gonna be "current year", then maybe we can also hardcode it.

Reply:
I think this is the FiscalYear the invoice belongs to so it might not be wise to hardcode it. While in most cases it should match the current year, there could be scenarios like in the first days of January where we could receive invoices from the previous year.

Reply:
Makes sense 👍